deno_dist/utils/jwt/jwt.ts

```diff
@@ -20,7 +20,7 @@ const decodeJwtPart = (part: string): unknown =>
   JSON.parse(utf8Decoder.decode(decodeBase64Url(part)))
 
 const param = (name: AlgorithmTypeName): AlgorithmParams => {
-  switch (name.toUpperCase()) {
```

Comment:
This is non-secure because no exact comparison is made 😇

Reply:
You are right!